agent/Dockerfile

```diff
@@ -0,0 +1,54 @@
+ARG version=4.3-4-jdk11
+
+FROM jenkins/inbound-agent:${version}
+
+ARG user=jenkins
+
+USER root
+
+    #Install Prerequisites
+RUN apt-get update && \
+	apt-get -y install \
```

Comment:
can you fix the mixed tabs and spaces in this file, might be best to add an editorconfig file to the repo

Reply:
Fixed fmt issue and added editorconfig file